browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -306,13 +306,14 @@ void RewardsDOMHandler::GetWalletProperties(const base::ListValue* args) {
 
 void RewardsDOMHandler::OnWalletInitialized(
     brave_rewards::RewardsService* rewards_service,
-    int error_code) {
+    int result) {
   if (!web_ui()->CanCallJavascript())
     return;
 
-  if (error_code == 0) {
+  // ledger::Result::WALLET_CREATED
+  if (result == 12) {
     web_ui()->CallJavascriptFunctionUnsafe("brave_rewards.walletCreated");
-  } else if (error_code != 3) {
+  } else if (result != 3 && result != 0) {
```

Comment:
is there a reason to not compare to `ledger::Result::<value>`? We seem to be doing so in other places outside of the native ledger.